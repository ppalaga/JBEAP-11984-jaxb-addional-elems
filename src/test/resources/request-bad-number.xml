<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:demo="http://jbeap11984.issues.wildfly.org/">
  <soapenv:Header />
  <soapenv:Body>
    <demo:hello>
      <arg0>Frank</arg0>
      <arg1>n</arg1>
    </demo:hello>
  </soapenv:Body>
</soapenv:Envelope>