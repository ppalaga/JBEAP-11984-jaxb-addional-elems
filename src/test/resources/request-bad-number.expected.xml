<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"><soap:Body><soap:Fault><faultcode>soap:Client</faultcode><faultstring>Unmarshalling Error: For input string: "n" </faultstring></soap:Fault></soap:Body></soap:Envelope>